src/media.css

```diff
@@ -65,6 +113,30 @@
   .m-md-reset {
     margin: 0 !important;
   }
+
+  .m-md-small {
```

Comment:
what does `md` indicate? i'm guessing the medium breakpoint. is there a way to describe this state without repeating sizes?

Reply:
`m-md-medium` is the one i have trouble with. does it expand to `margin-medium-medium`?

Reply:
So `md` in this case indicates a breakpoint which i realize overlaps with multipliers, now explicitly states `pos-tablet` since `mar` & `pad` change the same position multiplier